src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -137,6 +131,16 @@ function MoneyRequestPreviewContent({
         [transaction?.transactionID, transactionViolations],
     );
 
+    const hasDuplicates = duplicates.length > 0;
```

Comment:
Coming from https://github.com/Expensify/App/issues/52243, there is a mismatch on showing RBR on `ReportPreview` and in `MoneyRequestPreviewContent` because here we just check for the amount of the duplicates violation without checking whether the permission is enabled or not, which is different form `hasWarningTypeViolations` that used [here](https://github.com/Expensify/App/blob/5e69bfc36ddfd82a113606257abb93f2fa78f7db/src/components/ReportActionItem/ReportPreview.tsx#L160). And it fixed here https://github.com/Expensify/App/pull/52994